### ISSUE TITLE
Linux support

### ISSUE THREAD

**Issue by noaccOS** [Kaldaien/SoulsUnsqueezed#15]
On ```DS3T.exe``` launch it shows various errors (I think because it cannot read my GPU info) and when enabled I can't set my monitor's resolution when in game (which I could without the program). 
It plays at a resolution considerably lower.
Tested on ```Proton-GE-5.4-1```

**Comment by amadejkastelic** [comment]
@noaccOS Did you manage to get ultrawide working? Tried both this app and hex editor... Nothing worked.

**Comment by noaccOS** [comment]
> @noaccOS Did you manage to get ultrawide working? Tried both this app and hex editor... Nothing worked.

Nope, I played on windows and I got banned... Now I play on Linux without the mod

**Comment by Lightsockie** [comment]
This is one of the first places I found when I was trying to get ultrawide working, so gonna paste my protondb report here now that I have the "hex method" working in case others like me stumble here:

> Proton 5.0-9
> Distro: Arch Linux
> Kernel: 5.8.5-arch1-1
> RAM: 16 GB
> GPU Driver: 4.6 Mesa 20.1.6
> GPU: AMD Radeon RX 5700 XT
> CPU: AMD Ryzen 5 2600X Six-Core

> I got Ultrawide support working! Basically doing the "Hex Method", which is outlined here: https://steamcommunity.com/sharedfiles/filedetails/?id=668484112
> 
> Make sure you run the game once before you follow the guide, and set the game mode to windowed (if you don't/froget you can just edit the file to say WINDOW).
> 
> I actually made the GraphicsConfig.xml read-only too, but not sure if it was necessary.
> 
> I pieced together my resolution from these tables: https://www.wsgf.org/article/common-hex-values
> 
> 000F 0000 4006
> 
> 3840 x 1600
> 
> (The middle 4 values are always 0000 it seems)
> 
> The menus won't be ultrawide, but once you're ingame the window will expand like desired (and not stretch or show black bars)
> 
> Something very weird I experience is my GraphicsConfig.xml file having really weird artifacts/not-actually-display characters embedded in between every character. vim of course has no trouble with this and I could "cheat" the values outlined in the guide, but I imagine any other text editor wont' know how to read the characters (leafpad could only render a single <, for example).

(I know this doesn't further the scope of this ticket but I figure it's highly unlikely the dev cares about Linux users anyway lol)

----
Oh, to expand on the above, actually: I bet this program would have worked. My GraphicsConfig.xml file really is corrupted, and the DS3T.exe tool correctly recognized it as such. If someone didn't have a corrupted file (either cuz whatever thing is consistently happening to me across re-downloads doesn't effect them, or they find a clean file elsewhere), this could possibly get SouldsUnsqueezed working:
```
WINEPREFIX="~/.steam/steam/steamapps/compatdata/374320/pfx" wine DS3T.exe
```

**Comment by amadejkastelic** [comment]
@Lightsockie Thank you very much for your input. Finally figured it out. The problem was a strange encoding on my GraphicsConfig.xml. I removed it and created a new file with default settings copied from internet. Ultra wide is now working!
